contrib/ci/generate_dockerfile.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python3
+import os
+import sys
+
+directory = os.path.dirname(sys.argv[0])
+TARGET=os.getenv('OS')
+
+if TARGET == '':
+    print("Missing OS environment variable")
+    sys.exit(1)
+
+with open (os.path.join(directory,"dependency_matrix.csv"), 'r') as rfd:
```

Comment:
Not 100% about the name -- dependencies.txt or depends.txt might be better.

Reply:
OK.